README.md

```diff
@@ -102,3 +107,16 @@ This project is licensed under the MIT License - see the [LICENSE](LICENSE) file
 ## 🤝 Contributing
 
 Contributions are welcome! Feel free to open issues or submit pull requests to improve Ecoute.
+
+### Installation
+
+1. `make venv`
+2. Activate the venv: `ecout_venv`
```

Comment:
ecout_env\Scripts\activate ?

Reply:
I see, I'll add one script for MacOS and one for Windows.

Reply:
Ah, sorry. I tried adding a script, but it's not possible to activate the venv with a make command. I'll add both options to the README: for MacOS it's `source ecout_env/bin/activate`